src/components/Task.jsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable no-magic-numbers */
+import {
+  Card, Row, Col, Input, Select,
+} from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+
+import TaskDisplay from './TaskDisplay';
+
+const { Option } = Select;
+
+function Task(props) {
+  const {
+    handleMinutesChange,
+    handleOnDelete,
+    handleTaskTitleChange,
+    minutesChange,
+    task,
+    taskList,
+    titleChange,
+    totalScheduleTime,
+    totalTasksTime,
+  } = props;
+
+  const timeValues = [25, 30, 35, 45, 50, 60, 75, 90];
+
+  const timeOptionsDisplay = timeValues.map((value) => {
+    if (totalScheduleTime > value) {
+      if (totalScheduleTime - totalTasksTime > value) {
```

Comment:
A cleaner approach here would be putting the expression inside a variable. This way, you can decrease the cognitive load to other developers who will touch this code. The variable explains what the expression does. 

![image](https://user-images.githubusercontent.com/62787427/128623384-c6cb47af-7878-40cd-a575-264f77e5d148.png)

Reply:
Resolved